pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -344,3 +344,16 @@ def norm(input: Tensor,
 
 def norm_except_dim(v: Tensor, pow: int = 2, dim: int = 0) -> Tensor:
   return modules.Norm(p=pow, axes=[i for i in range(v.dim()) if i != dim], keepdims=True)(v)
+
+
+def group_norm(input: Tensor, num_groups: int, weight: Optional[Tensor] = None, bias: Optional[Tensor] = None,
+               eps: float = 1e-5) -> Tensor:
+  module = modules.GroupNorm(num_groups=num_groups, num_channels=0, eps=eps, affine=False)
+  out = module.as_returnn_torch_functional()(input)
+  assert out.shape[1] == weight.shape[0], "data should be of shape (B, F, *) and weights should be of shape (F,)"
```

Comment:
Only if `weight is not None`. So move it to the branch below.
Also directly check the full `shape`. I.e.
```suggestion
  assert (out.shape[1],) == weight.shape
```
Same also for `bias`.